er_physical.puml

```diff
@@ -66,12 +68,12 @@ entity Trainer{
     * Firstname : VARCHAR(50)
     * Lastname : VARCHAR(50)
     * BirthDate : DATE
-    NationnalityId : INTEGER <<FK>>
+    NationalityId : INTEGER <<FK>>
     -- constraints --
     CHECK (BirthDate > 1950)
```

Comment:
Use YEAR(BirthDate), otherwise SQL server will cast your date to the internal numeric format, which is not the year.